### ISSUE TITLE
Forcing.location returns an error

### ISSUE THREAD

**Issue by SarahAlidoost** [eWaterCycle/ewatercycle#50]
when I run the command as:

```python
from ewatercycle import forcing
from esmvalcore.experimental import CFG

CFG.load_from_file('~/.esmvaltool/config-user.yml')

forcing_output = forcing.generate(
    model='lisflood', 
    dataset='ERA-Interim',
    startyear=1990, 
    endyear=1990,
    shapefile='Meuse/Meuse.shp',
)

forcing_output.location
```

it returns the following error: 
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-9-f057557240f3> in <module>
----> 1 forcing_output.location

~/miniconda3/envs/ewatercycle/lib/python3.9/site-packages/ewatercycle/forcing/forcing_data.py in location(self)
      7     def location(self):
      8         """Return forcing directory."""
----> 9         return self.recipe_output.session.session_dir

AttributeError: 'RecipeOutput' object has no attribute 'session'
```

**Comment by SarahAlidoost** [comment]
@stefsmeets please see my comment here, any idea what is going on?

**Comment by stefsmeets** [comment]
> @stefsmeets please see my comment here, any idea what is going on?

Can you check the version of esmvalcore/esmvaltool that you are using? This is a relatively new addition that should be available in the latest release (2.2).

**Comment by SarahAlidoost** [comment]
> > @stefsmeets please see my comment here, any idea what is going on?
> 
> Can you check the version of esmvalcore/esmvaltool that you are using? This is a relatively new addition that should be available in the latest release (2.2).

Here, the versions of esmvalcore and esmvaltool in my ewatercycle conda environment:
```
esmvalcore                2.2.0           
esmvaltool-python         2.2.0
```

**Comment by stefsmeets** [comment]
Just debugged this with @SarahAlidoost , seems like this functionality was added in https://github.com/ESMValGroup/ESMValCore/pull/991 which will be available in the next release of esmvaltool (2.3) on 2021-06-28.
